### ISSUE TITLE
Client crashing often

### ISSUE THREAD

**Issue by Eufranio** [HexagonMC/BungeeCord#92]
Sometimes, when a player switch server from a Modded one to a Spigot one, he crashes with the following crash-report:
http://hastebin.com/ahiliyaqaq.sql

Is it related to Spigot or Bungee?

**Comment by i9hdkill** [comment]
Any error logs spigot or bungeeside while the crahs happens?
Which HexaCord version?

**Comment by Eufranio** [comment]
There are no errors in the log/bungee console when the crash happens. I'm using HexaCord v98.

**Comment by i9hdkill** [comment]
Could you test 99 or 96?

**Comment by Eufranio** [comment]
I don't know when this happens, updated to 99 and waiting...

**Comment by Eufranio** [comment]
Tested in this 2 versions, happened in both.

**Comment by i9hdkill** [comment]
Any chance that you can figure out, how to reproduce this?

**Comment by Eufranio** [comment]
Nope, I've tested a lot of stuff. The issue seems to be only Modded -> Spigot, not Spigot -> Modded...

**Comment by LemADEC** [comment]
Logs are gone. Is it still a thing or could it be closed?
